Mirror/Runtime/NetworkProximityChecker.cs

```diff
@@ -31,6 +31,14 @@ public enum CheckMethod
 
         float m_VisUpdateTime;
 
+        // OverlapSphereNonAlloc array to avoid allocations.
+        // -> static so we don't create one per component
```

Comment:
The fields are not static.   Did you mean to make them static?

Reply:
I did, thanks